Add doc for helm CLI 3.7 (#258)
goharbor/website: docs/working-with-projects/working-with-images/managing-helm-charts.md

## Manage Helm Charts with the OCI-compatible registry of Harbor

> ### If the version of Helm CLI you are using is before 3.7.0, please refer to the following

Reviewer: ```suggestion
### If the Helm CLI version is lower than 3.7.0, please refer to the following
```

```sh
helm chart pull xx.xx.xx.xx/library/dummy-chart:version
```

> ### If the version of Helm CLI you are using is 3.7.0 or later, please refer to the following

Reviewer: ```suggestion
### If the Helm CLI version is 3.7.0 or higher, please refer to the following
```

Helm 3 now supports registry operations for an OCI-compatible registry including pushing and pulling. To install the latest Helm CLI, please refer [install helm](https://helm.sh/docs/intro/install/). Please also run `helm version` command to make sure the version of Helm CLI is v3.7.0+.

```sh
helm version

#version.BuildInfo{Version:"v3.7.0", GitCommit:"eeac83883cb4014fe60267ec6373570374ce770b", GitTreeState:"clean", GoVersion:"go1.16.8"}
```

### Login to the OCI-compatible registry of Harbor

> Before pull/push helm charts with the OCI-compatible registry of Harbor, Harbor should be logged with `helm registry login` command.

Reviewer: ```suggestion
Prior pull/push helm charts with the OCI-compatible registry of Harbor, should "login" by `helm registry login` command.
```
